datasets/newsqa/newsqa.py

```diff
@@ -0,0 +1,326 @@
+                                    "sourcerAnswers": datasets.features.Sequence(
+                                        {
+                                            "s": datasets.Value("int32"),
+                                            "e": datasets.Value("int32"),
+                                            "noAnswer": datasets.Value("bool"),
+                                        }
+                                    ),
+                                }
+                            ),
+                            "validated_answers": datasets.features.Sequence(
+                                {
+                                    "sourcerAnswers": datasets.features.Sequence(
+                                        {
+                                            "s": datasets.Value("int32"),
+                                            "e": datasets.Value("int32"),
+                                            "noAnswer": datasets.Value("bool"),
+                                            "count": datasets.Value("int32"),
+                                        }
+                                    ),
+                                }
+                            ),
+                        }
+                    ),
+                }
+            )
+        else:  # This is the name of the configuration selected in BUILDER_CONFIGS above
+            features = datasets.Features(
+                {
+                    "story_id": datasets.Value("string"),
+                    "story_text": datasets.Value("string"),
+                    "question": datasets.Value("string"),
+                    "answer_token_ranges": datasets.Value("string"),
+                }
+            )
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=features,  # Here we define them above because they are different between the two configurations
+            # If there's a common (input, target) tuple from the features,
+            # specify them here. They'll be used if as_supervised=True in
+            # builder.as_dataset.
+            supervised_keys=None,
+            # Homepage of the dataset for documentation
+            homepage=_HOMEPAGE,
+            # License for the dataset if available
+            license=_LICENSE,
+            # Citation for the dataset
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+
+        path_to_manual_folder = os.path.abspath(os.path.expanduser(dl_manager.manual_dir))
+        combined_file_csv = path_to_manual_folder + "/combined-newsqa-data-v1.csv"
+        combined_file_json = path_to_manual_folder + "/combined-newsqa-data-v1.json"
+        split_files = path_to_manual_folder + "/split_data/"
+
```

Comment:
Can you check that the expected files exist in the path_to_manual_folder ?
You can do the same as in [matinf](https://github.com/huggingface/datasets/blob/master/datasets/matinf/matinf.py#L140-L145)